runtime/compiler/control/CompilationThread.cpp

```diff
@@ -10147,14 +10147,13 @@ TR::CompilationInfoPerThreadBase::compile(
          omrthread_jit_write_protect_disable();
          // Put a metaData pointer into the Code Cache Header(s).
          //
-         uint8_t *warmMethodHeader = compiler->cg()->getBinaryBufferStart() - sizeof(OMR::CodeCacheMethodHeader);
-         memcpy( warmMethodHeader + offsetof(OMR::CodeCacheMethodHeader, _metaData), &metaData, sizeof(metaData) );
+         OMR::CodeCacheMethodHeader *warmMethodHeader = reinterpret_cast<OMR::CodeCacheMethodHeader*>(compiler->cg()->getBinaryBufferStart() - sizeof(OMR::CodeCacheMethodHeader));         
+         warmMethodHeader->_metaData = metaData;
```

Comment:
Suggest rather than repeating `OMR::CodeCacheMethodHeader` in `sizeof()`, use `- 1` after the `reinterpret_cast<>()`:
```c
         OMR::CodeCacheMethodHeader *warmMethodHeader = reinterpret_cast<OMR::CodeCacheMethodHeader *>(compiler->cg()->getBinaryBufferStart()) - 1;
```
Likewise for line 10154, but please be consistent in formatting pointer types (space before, but not after `*`).